### ISSUE TITLE
strange behavior rule `declaration-colon-newline-after`

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#896]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

See example

> Which rule, if any, is this issue related to?

`declaration-colon-newline-after`

> What CSS is needed to reproduce this issue?

```scss
// Valid
a {
    box-shadow:
        0 0 0 1px #5b9dd9,
        0 0 2px 1px rgba(30, 140, 190, 0.8);
}

// Valid
a {
    box-shadow:
        0 0 0 1px #5b9dd9, 0 0 2px 1px rgba(30, 140, 190, 0.8);
}

// Invalid, but error `Unexpected whitespace after ":" with a multi-line value (declaration-colon-newline-after)`, although it was expected to see an error about require a newline
a {
    box-shadow: 0 0 0 1px #5b9dd9,
        0 0 2px 1px rgba(30, 140, 190, 0.8);
}

//Invalid, no errors, but expected error about require a newline after `:`, because it is multiline declaration.
a {
    box-shadow:0 0 0 1px #5b9dd9,
        0 0 2px 1px rgba(30, 140, 190, 0.8);
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "declaration-colon-newline-after": "always-multi-line"
  }
}
```

> Which version of stylelint are you using?

`4.5.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

Throw error about require newline.

> What actually happened (e.g. what warnings or errors you are getting)?

Nothing

**Comment by davidtheclark** [comment]
Fixed released in v5.
